frontend/src/components/worksheets/BundleDetail/MainContent.jsx

```diff
@@ -148,6 +142,7 @@ const styles = (theme) => ({
         flexWrap: 'wrap', 
         flexShrink: 1,
         overflow:'auto',
+        whiteSpace: 'pre-line',

```

Comment:
This will let the div block respect '\n' character and break new lines for stdout & stderr